buildpipeline/linux.arm64.groovy

```diff
@@ -41,8 +41,9 @@ simpleDockerNode('microsoft/dotnet-buildtools-prereqs:ubuntu-16.04-cross-arm64-a
     stage ('Build Packages') {
         sh "./build-packages.sh -buildArch=arm64 -${params.CGroup}"
     }
-
-    // TODO: Build Tests for arm64 when possible
+    stage ('Build Tests') {
+        sh "./build-tests.sh -buildArch=arm64 -${params.CGroup}"
```

Comment:
I expect that you will need to have this buildArch match what is being used in the `Build Managed` step which is x64. @sdmaclea correct me if I'm wrong but IIRC we used x64 because of some toolset issue with arm64 correct?

Reply:
arm64 cli has not been released for .net core.  So building for x64 was adequate for arm64.